{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/wsgi.py

```diff
@@ -17,6 +15,9 @@
 from decouple import config
 from whitenoise.django import DjangoWhiteNoise
 
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', '{{ cookiecutter.project_name }}.settings')
```

Comment:
I think this line has to before `from django.conf import settings`. `django-admin startproject` generates `wsgi.py` this way.